dev/io.openliberty.org.apache.jasper.expressionLanguage.5.0_fat/test-applications/EL50BasicTests.war/src/io/openliberty/el50/fat/basic/servlets/EL50BasicTestsServlet.java

```diff
@@ -0,0 +1,167 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.el50.fat.basic.servlets;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import componenttest.app.FATServlet;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import jakarta.el.ELContext;
+import jakarta.el.ELManager;
+import jakarta.el.ELProcessor;
+import jakarta.el.ELResolver;
+import jakarta.el.ExpressionFactory;
+import jakarta.el.StandardELContext;
+import jakarta.el.ValueExpression;
+import jakarta.servlet.annotation.WebServlet;
+
+
+/**
+ * Servlet for testing basic changes in EL 5.0
+ */
+@WebServlet({ "/EL50BasicTests" })
+public class EL50BasicTestsServlet extends FATServlet {
+    private static final long serialVersionUID = 1L;
+
+    ELProcessor elp;
+
+    public EL50BasicTestsServlet() {
+        super();
+
+        elp = new ELProcessor();
+        elp.defineBean("simpleBean", new SimpleBean());
+    }
+
+    /**
+     * 
+     * Expression Language 5.0 in Jakarta EE10 added generics to methods in multiple classes https://github.com/jakartaee/expression-language/issues/157
+     * Classes with changes we tests are ELProcessor, ExpressionFactory, ELContext, ValueExpression, ELResolver (returns null)
+     * Classes with changes we don't test TypeConverter(returns null)
+     * Classes with changes we implicitly test CompositeELResolver(returns null) since it is used by StandardELContext internally
+     *
+     * @throws Exception
+     */
+    @Test
+    @Mode(TestMode.FULL)
+    public void testGenerics() throws Exception {
+        // Test generics with ELProcessor uses ValueExpression.getValue internally that uses generics
+        // No need to cast this to boolean like previous versions
+        assertTrue("The type was expected to be true for ELProcessor but was not", elp.eval("simpleBean.getNumber() == 25") instanceof Boolean);
```

Comment:
I think I was wrong recommending `instanceof`. You should remove `instanceof Boolean`

passing it directly into assertTrue is better as you mentioned (since assertTrue expects a boolean, not object)

Reply:
I'll make the change